hedera-node/hedera-networkadmin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/handlers/FreezeHandler.java

```diff
@@ -72,23 +73,23 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
 
         final FreezeType freezeType = freezeTxn.freezeType();
         final ReadableSpecialFileStore specialFileStore = context.createStore(ReadableSpecialFileStore.class);
+        final TransactionID txnId = context.body().transactionID();
+        assert (txnId != null);
```

Comment:
Do we want this native java language assert, or do we want to throw a custom exception? I can go either way, this just struck me as odd for some reason I can't quite put my finger on

Reply:
It is a bit odd that I'm using assert(_ != null) here and requireNonNull(_) above. I only put the check in there to quiet a SonarLint warning. I don't know how it would ever get to this point in the transaction handling and be null.
I will change it to requireNonNull(_)